includes/manage_pages.inc

```diff
@@ -794,6 +801,12 @@ function islandora_paged_content_add_pages($repository, $config, $pages_director
   $parent = $config['pid'];
   $object->relationships->add(FEDORA_MODEL_URI, 'hasModel', $config['model']);
 
+  // Re-checking module existence here in case this batch op is called from
+  // somewhere other than the form.
+  if (module_exists('islandora_ocr')) {
+    module_load_include('inc', 'islandora_ocr', 'includes/derivatives');
+    islandora_ocr_set_generating_rels_ext_statements($object, !$config['ignored_derivs']['ocr'], !$config['ignored_derivs']['hocr']);
+  }
   islandora_paged_content_set_relationship($rels_ext, ISLANDORA_RELS_EXT_URI, 'hasLanguage', $config['language'], TRUE);
```

Comment:
Since "hasLanguage" is part of the OCR piece, should it go into islandora_ocr_set_generating_rels_ext_statements?

Reply:
I'd agree with that!